pkg/operator/controller/ingress/status_test.go

```diff
@@ -1312,7 +1312,12 @@ func TestComputeIngressUpgradeableCondition(t *testing.T) {
 					},
 				},
 			}
-			wantSvc, service, err := desiredLoadBalancerService(ic, deploymentRef, platformStatus)
+			networkConfig := &configv1.Network{
+				Spec: configv1.NetworkSpec{
+					NetworkType: "OpenShiftSDN",
+				},
+			}
```

Comment:
Normally we would add a new variable to the test case itself, and add a couple of test cases to test the variant values.